docs/ref/migration-operations.txt

```diff
@@ -195,16 +195,6 @@ if the field is nullable or if it has a default value that can be used to
 populate the recreated column. If the field is not nullable and does not have a
 default value, the operation is irreversible.
 
-.. admonition:: PostgreSQL
```

Comment:
I think we want a `.. versionchanged:: 5.2` block here to explain that `CASCADE` is no longer used in the base backend and PostgreSQL.